### DISCUSSION TITLE
release  is  too  large

### DISCUSSION THREAD

**Discussion by 3052** [wezterm/wezterm#6640] [Q&A]
I am coming from Windows terminal, which  is currently 11 MB:

https://github.com/microsoft/terminal/releases/tag/v1.20.11781.0

but WezTerm is 63 MB:

https://github.com/wez/wezterm/releases/tag/20240203-110809-5046fc22

why  the big difference?

**Comment by bew** [comment]
Might be because Wezterm embeds its default font and an emoji font, for a uniform default rendering across the platforms 🤔

**Reply by 3052** [reply]
not worth it.

**Comment by tjex** [comment]
@3052 This is most definitely a discussion topic and should not add to the list of issues that needs to be scanned over to find actual issues!

Please move there.
